packages/vanilla/src/components/global-nav/side-nav/side-nav-compact/side-nav-compact.scss

```diff
@@ -0,0 +1,17 @@
+.hig__global-nav__sidenav-compact {
```

Comment:
Icons should be white when the theme is dark-blue. Check the other details too.

Reply:
:+1: Gemini test wasn't programmatically setting the theme to the Group. Will fix before merge.